### ISSUE TITLE
New "memory" parameter sneaking through from sklearn to user interface

### ISSUE THREAD

**Issue by BenKaehler** [qiime2/q2-feature-classifier#101]
There is a new `memory` parameter to [`sklearn.pipeline.Pipeline`](http://scikit-learn.org/stable/modules/generated/sklearn.pipeline.Pipeline.html) that is sneaking through to the user interface. It's not useful to users and is already causing [confusion](https://forum.qiime2.org/t/plugin-error-from-feature-claasifier/3759).

I suggest masking it out in `classify._pipeline_signature`, but I haven't thought about it too thoroughly yet.

**Comment by nbokulich** [comment]
another [forum xref](https://forum.qiime2.org/t/plugin-error-from-feature-claasifier/3759?u=nicholas_bokulich)
